scss/components/button-group.scss

```diff
@@ -51,3 +51,13 @@ $block-alt: #{$fd-namespace}-segmented-button;
     }
   }
 }
+.fd-button-group .fd-button--compact:hover[aria-disabled=true],
+.fd-button-group .fd-button:hover[aria-disabled=true] {
```

Comment:
I'm not sure if `aria-disabled` is the best selector to use here.  Looking through other examples, I see buttons are using an `is-disabled` class and in another library we use, we are using the `disabled` HTML attribute.  I think the `disabled` HTML attribute is probably going to be the best option going forward since there will be other a11y benefits by using the HTML spec.

The consuming library may also add the `aria-disabled` attribute, but we shouldn't base styling upon it.

Reply:
I totally agree. Thank you for pointing it out.